contracts/libraries/Pool.sol

```diff
@@ -108,10 +116,10 @@ library Pool {
     function initialize(
         State storage self,
         uint160 sqrtPriceX96,
-        uint8 protocolSwapFee,
-        uint8 hookSwapFee,
-        uint8 protocolWithdrawFee,
-        uint8 hookWithdrawFee
+        uint16 protocolSwapFee,
+        uint16 hookSwapFee,
+        uint16 protocolWithdrawFee,
+        uint16 hookWithdrawFee
```

Comment:
I think we can make the interface for this function

`initialize(State storage self, uint160 sqrtPriceX96, uint24 protocolFee, uint24 hookFee)`